test/resources/case38_operator_install/operator-policy-no-group-one-version.yaml

```diff
@@ -22,5 +22,8 @@ spec:
     sourceNamespace: olm
     startingCSV: quay-operator.v3.10.0
   versions:
-    - quay-operator.v3.10.0
+    - '{{ if false }}some value to test empty string is discarded{{ end }}'
+    - '{{ if true }}quay-operator.v3.10.0{{ end }}'
+    - '{{ if false }}some other value to test empty string is discarded{{ end }}'
+    - '{{ if false }}some other value to test empty string is discarded{{ end }}'
```

Comment:
These two are the same, Is this intended?

Reply:
It was just to further test the code of removing empty strings.